src/components/Popover/index.tsx

```diff
@@ -117,17 +123,40 @@ export default function Popover({
   const updateCallback = useCallback(() => {
     update && update()
   }, [update])
-  useInterval(updateCallback, show ? 100 : null)
+  useInterval(updateCallback, show ? 200 : null)
+
+  useEffect(() => {
+    if (delayBeforeShow) {
```

Comment:
prefer guard statement here, so:

```
if (!delayBeforeShow) return 

...
```

and then you can avoid the nested if statements

Reply:
makes sense ty done